demo/Views/DateTimePickerView.vala

```diff
@@ -78,23 +102,31 @@ public class DateTimePickerView : Gtk.Grid {
         set_selected_datetime ();
         datepicker.changed.connect (() => set_selected_datetime ());
         timepicker.changed.connect (() => set_selected_datetime ());
+        timepicker_with_seconds.changed.connect (() => set_selected_datetime_with_seconds ());
+
+        
 
         column_spacing = 12;
-        row_spacing = 6;
+        row_spacing = 9;
```

Comment:
Can you revert this spacing change please?

Reply:
Doh...at one point I thought this was the number of rows in the grid for some reason, and forgot that I had changed it under that wrong impression. Done.